litestar/cli/_utils.py

```diff
@@ -114,7 +115,10 @@ def from_env(cls, app_path: str | None, app_dir: Path | None = None) -> Litestar
             import dotenv
 
             dotenv.load_dotenv()
+
         app_path = app_path or getenv("LITESTAR_APP")
+        if app_path and getenv("LITESTAR_APP") is None:
```

Comment:
I think so because both `app_path` and `LITESTAR_APP` can be `None`.  Additionally,  I think if the `app_path` is already set, it will never evaluate the `getenv("LITESTAR_APP")`, correct?

Reply:
Aah yeah that makes sense